test/regress/regress-37.txt

```diff
@@ -0,0 +1,40 @@
+;;; TOOL: run-objdump
+;;; ARGS0: -v --enable-bulk-memory
```

Comment:
I'm not a big fan of these regress tests.  Wouldn't it be better to distil them into something more specific with a meaningful name?   Perhaps you are following some pattern that I'm not aware of?   Is there good reason for such a practice over just writing new unit tests?

Reply:
Yeah, you're right. They're fine for tests that are odd combinations, but probably shouldn't be added when the real test is natural.